### PR TITLE
Adex dataset

### PR DESCRIPTION
**Pull request by jeetendersinghc**

### CODE REVIEW

**Comment by elong0527** [comment]
Let's enhance the readability of this example.

**Review by nanxstats — CHANGES_REQUESTED** [review]
General:

- Consider writing an [informative title and description](https://code-review.tidyverse.org/author/submitting.html) for the pull request (also the commit messages).

File structure:

- As this is the first time `data-raw/` is added, add it to `.Rbuildignore`.
- To follow the convention, use an `.R` file instead of using an `.Rmd` file.

Content:

- Minimize the namespaces used and only load the essential ones. For example, why is glue needed? Please avoid `library(tidyverse)` because it will unnecessarily inflate the number of dependencies required to run this script.
- Follow a consistent style in writing file and directory paths. For example, the leading `./` and `../` are redundant if the project is already opened.
- For code comments, please follow the style defined in the [tidyverse style guide](https://style.tidyverse.org/functions.html#comments-1) and make sure the capitalization of words are correct.
- Run `styler::style_file()` to apply the tidyverse code style.
- Run lintr on the code. Things like `# getwd()` will be marked as "commented out code should be removed".
- All exported datasets should be [documented](https://r-pkgs.org/data.html#sec-documenting-data).

File: data-raw/adex.Rmd [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+# Merge both dataset by usubjid
+ ex1<- merge(adsl, ex0, by =c("STUDYID", "USUBJID"))  
+
+```
+
+
+
+```{r}
+# Derive ASTDT and AENDT
+adex1 <-ex1 %>% mutate(ASTDT = as.Date(EXSTDTC, format='%Y-%m-%d')) %>% mutate(AENDT = as.Date(EXENDTC, format='%Y-%m-%d'))
```
Comment (wangben718) [inline data-raw/adex.Rmd:51]: run `lintr` to style the code.

**Comment by elong0527** [comment]
The PR has opened > 30 days, tentatively close it. Please create a new PR once comments are addressed.

**Pull request closed without merging**